portal-impl/src/META-INF/security-spring.xml

```diff
@@ -20,6 +20,7 @@
 	<bean class="com.liferay.portal.kernel.security.auth.DefaultFullNameValidator" id="com.liferay.portal.kernel.security.auth.DefaultFullNameValidator" />
 	<bean class="com.liferay.portal.kernel.security.auth.DefaultScreenNameGenerator" id="com.liferay.portal.kernel.security.auth.DefaultScreenNameGenerator" />
 	<bean class="com.liferay.portal.kernel.security.auth.DefaultScreenNameValidator" id="com.liferay.portal.kernel.security.auth.DefaultScreenNameValidator" />
+	<bean class="com.liferay.portal.security.auth.tunnel.TunnelAuthenticationManagerImpl" id="com.liferay.portal.kernel.security.auth.tunnel.TunnelAuthenticationManager" />
```

Comment:
@W2jx It seems the beans are sorted based on their classes? It won't work if we keep it at line 86?

Reply:
Hi @dantewang, SF asks us to sort these beans based on bean id.

Reply:
Hi @dantewang , yes, it will work, the id can be any string, as long as it is unique

Reply:
@W2jx So the order in the xml file doesn't matter. Let's keep the original id, and let the util use it.

Reply:
Ok, I will resend it.